lib/engine/game/g_18_india/game.rb

```diff
@@ -1230,6 +1230,222 @@ def price_movement_chart
             ['Dividend ≥ 4x share price', '4 →'],
           ]
         end
+
+        # Modifies the look of Part::largeIcons (controls the how they are decorated)
+        def decorate_marker(_icon)
+          { color: '', shape: :hexagon }
+        end
+
+        # Map Legends for commodity and connection bounus
+
+        def show_map_legend?
+          true
+        end
+
+        def show_map_legend_on_left?
+          false
+        end
+
+        def map_legends
+          %i[commodity_legend connection_legend]
+        end
+
+        def connection_legend(_font_color, _yellow, green, _brown, _gray, _red, action_processor: nil)
+          cell_style = {
+            border: '1px solid',
+            color: 'black',
+            'font-weight': 'bold',
+            'text-align': 'center',
+            'vertical-align': 'middle',
+            height: '33px',
+          }
+
+          [
+            # table-wide props
+            {
+              style: {
+                margin: '0.5rem 0 0.5rem 0',
+                border: '1px solid',
+                borderCollapse: 'collapse',
+              },
+            },
+            [
+              { text: 'Connection Bonus', props: { attrs: { colspan: 10 }, style: { **cell_style, backgroundColor: green } } },
+            ],
+            [
+              { text: 'Delhi (G8) <=> Kocchi (G36)', props: { style: cell_style } },
+              { text: format_currency(100), props: { style: cell_style } },
+            ],
+            [
+              { text: 'Karachi (A16) <=> Chennai (K30)', props: { style: cell_style } },
+              { text: format_currency(80), props: { style: cell_style } },
+            ],
+            [
+              { text: 'Lahore (D3) <=> Kolkata (P17)', props: { style: cell_style } },
+              { text: format_currency(80), props: { style: cell_style } },
+            ],
+            [
+              { text: 'Nepal (M10) <=> Mumbai (D23)', props: { style: cell_style } },
```

Comment:
Unicode support should be universal these days, so we don't have to use Ascii compositions for arrows:
```suggestion
              { text: 'Delhi (G8) ⟷ Kocchi (G36)', props: { style: cell_style } },
              { text: format_currency(100), props: { style: cell_style } },
            ],
            [
              { text: 'Karachi (A16) ⟷ Chennai (K30)', props: { style: cell_style } },
              { text: format_currency(80), props: { style: cell_style } },
            ],
            [
              { text: 'Lahore (D3) ⟷ Kolkata (P17)', props: { style: cell_style } },
              { text: format_currency(80), props: { style: cell_style } },
            ],
            [
              { text: 'Nepal (M10) ⟷ Mumbai (D23)', props: { style: cell_style } },
```